<launch>

	<node pkg="course_agv_slam_task" type="mapping" name="mapping" output="screen">
		
		<!--from a rosbag-->
		<param name="world_frame" value="map"/>
		<!-- <param name="sensor_frame" value="ekf_slam"/> -->
		<param name="sensor_frame" value="robot_base"/>


		<param name="map_height" value="129"/>
		<param name="map_width" value="129"/>
		<param name="map_res" value="0.155"/>
		<param name="sensor_range" value="14.5"/>
		<param name="prob_init" value="0.5"/>
		<param name="prob_free" value="0.1"/>
		<param name="prob_occu" value="0.995"/>
		<param name="prob_thres" value="0.8"/>


	</node>

    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find course_agv_nav)/launch/nav.rviz"/> -->
</launch>
